src/components/molecules/SearchAutocomplete/suggestionsList.tsx

```diff
@@ -17,14 +19,23 @@ const SuggestionsList: FC<SuggestionsProps> = ({
       aria-labelledby="search-input-label"
       role="listbox"
     >
-      {filteredSuggestions.map((suggestion: string, index: number) => (
+      {filteredSuggestions.map((suggestion, index) => (
         <S.Suggestion
-          key={suggestion}
+          key={suggestion.id}
```

Comment:
We should style differently suggestion for mobile devices - https://www.figma.com/file/FZKtRjUFs3QHXZgQKnJRMW/Thinking-Classroom?node-id=891%3A0  

We could do this also in different task.

Reply:
let's do it separately :)

Reply:
Yes, it is for tablets. I've created a new task in Asana :)